extensions/lifecycle/objectProcessor/LifecycleObjectTransitionProcessor.js

```diff
@@ -48,8 +49,32 @@ class LifecycleObjectTransitionProcessor extends LifecycleObjectProcessor {
         return 'transition-processor';
     }
 
+    _getTransitionTaskConsumerParams() {
```

Comment:
should be refactored to avoid duplicating the actual "definition" of the params

maybe introduce a function `getConsumerParam(topic)` in ObjectProcessor, or just add an (optional) param to `getConsumerParams(topic = this._lcConfig.objecTasksTopic)`...